apps/nextjs-website/src/components/organisms/Auth/ResetPasswordSuccess.tsx

```diff
@@ -43,20 +41,26 @@ const ResetPasswordSuccess = ({
               <IllusEmailValidation />
             </Stack>
             <Typography variant='h4' pt={5} mb={4} textAlign='center'>
-              {resetPassword.checkEmailTitle}
-            </Typography>
-            <Typography variant='body1' mb={2}>
-              {resetPassword.checkEmail(email)}
+              {resetPassword('checkEmailTitle')}
             </Typography>
+            <Typography
+              variant='body1'
+              mb={2}
+              dangerouslySetInnerHTML={{
+                __html: resetPassword
+                  .raw('checkEmail')
+                  .replace('{email}', email),
```

Comment:
I think is here is better to use [parametric translation](https://next-intl-docs.vercel.app/docs/usage/messages) of `next-intl` instead of replace